package.json

```diff
@@ -46,10 +46,9 @@
   "dependencies": {
     "bluebird": "^2.9.32",
     "ext2fs": "0.0.8",
-    "fatfs": "^0.10.3",
+    "fatfs": "https://github.com/zvin/fatfs/tarball/error-event-if-init-fails",
```

Comment:
Doesn't npm support `git://` URLs pointing at a specific branch? (at least, I'm fairly sure I've seen something like that used elsewhere)
Would that be better than a `https://` URL pointing to a tarball? \*shrug\*